doc/sphinx/source/project/introduction.rst

```diff
@@ -4,189 +4,97 @@
 
-Problem: Absence of graph-level optimizations
+Problem 1: Absence of graph-level optimizations
 ---------------------------------------------
 
-The diagram below illustrates a simple example of how a deep learning 
-framework, when integrated with a kernel library, is capable of running each 
-operation in a computational graph optimally, but the graph itself may not be 
-optimal: 
+The example diagram below shows how a deep learning framework, when integrated with a kernel library, can optimally run each operation in a computational graph, but the graph itself may not be optimal.
 
 .. _figure-A:
 
 .. figure:: ../graphics/intro_graph_optimization.png
    :width: 555px
    :alt: 
 
-The computation is constructed to execute ``(A+B)*C``, but in the context of 
-nGraph, we can further optimize the graph to be represented as ``A*C``. From the 
-first graph shown on the left, the operation on the constant ``B`` can be 
-computed at the compile time (known as constant folding), and the graph can be 
-further simplified to the one on the right because the constant has value of 
-zero. Without such graph-level optimizations, a deep learning framework with a 
-kernel library will compute all operations, and the resulting execution will be 
-suboptimal. 
+The computation is constructed to execute ``(A+B)*C``, but in the context of nGraph, we can further optimize the graph to be represented as ``A*C``. From the first graph shown on the left, the operation on the constant ``B`` can be computed at the compile time (known as constant folding), and the graph can be further simplified to the one on the right because the constant has a value of zero. Without such graph-level optimizations, a deep learning framework with a kernel library will compute all operations, resulting in suboptimal execution. 
 
 
-Problem: Reduced scalability 
+Problem 2: Reduced scalability 
 ----------------------------
 
-Integrating kernel libraries with frameworks is increasingly becoming 
-nontrivial due to the growing number of new deep learning accelerators. 
-For each new deep learning accelerator, a custom kernel library integration 
-must be implemented by a team of experts. This labor-intensive work is 
-further amplified if you want your DL accelerator to support a number of 
-different frameworks. The work must be revisited any time you upgrade or 
-expand your network's hardware. Each integration is unique to the framework 
-and its set of deep learning operators, its view on memory layout, its 
-feature set, etc.
+Due to the growing number of new deep learning accelerators, integrating kernel libraries with frameworks has become increasingly nontrivial. For each new deep learning accelerator, a custom kernel library integration must be implemented by a team of experts. This labor-intensive work is further complicated by the number of frameworks as illustrated in the diagram below using orange lines. 
 
+.. _figure-B:
+
+.. figure:: ../graphics/lorem-ipsum.png
```

Comment:
Is this file present in the repo? (If so, maybe we should rename it from `lorem-ipsum.png` :))